integration_tests/tests/triggers/trigger_rollback.rs

```diff
@@ -30,7 +31,12 @@ fn failed_trigger_revert() -> Result<()> {
     let _ = client.submit_blocking(register_trigger);
 
     let call_trigger = ExecuteTrigger::new(trigger_id);
-    client.submit_blocking(call_trigger)?;
+    let _err = client
+        .submit_blocking(call_trigger)
+        .expect_err("should immediately result in error")
+        .root_cause()
+        .downcast_ref::<InstructionExecutionError>()
+        .expect("unexpected error");
```

Comment:
If possible, would be great to assert the exact contents of the error. For example, for me as a reviewer, it is not clear what kind of `InstructionExecutionError` is expected here.